Azure-Samples/Cognitive-Services-Voice-Assistant
# Fix for delete profile bug

## Purpose
* Once a profile is deleted, the profile name is set to the next profile in the list and configuration values are of the associated profile. 
*In the previous version, deleting a profile resulted in the profile name to be blank along with blank configuration values.

## Does this introduce a breaking change?
[ ] Yes
[x] No

## Pull Request Type
What kind of change does this Pull Request introduce?
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:

## How to Test
* Create several profiles
* Select one and delete it
* Profile name should no longer be blank, but instead of profile index 0
